src/gui/test/screen_menu_test.hpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file screen_menu_test.hpp
+ * @brief test screen, accessible in debug only
+ */
+
+#pragma once
+
+#include "screen_test_gui.hpp"
+#include "screen_test_term.hpp"
+#include "screen_test_msgbox.hpp"
+#include "screen_qr_error.hpp"
+#include "screen_test_wizard_icons.hpp"
+#include "screen_test_dlg.hpp"
+#include "screen_menu_eeprom_test.hpp"
+#include "screen_test_load.hpp"
+
+//#include "menu_opener.hpp" TODO make it work
```

Comment:
TODO left

Reply:
It currently does not work
I plan to fix it later
It is test menu, which is only in debug